frontend/src/components/login.js

```diff
@@ -1,10 +1,35 @@
 import React from 'react';
 
-const Login = () => (
-    <div>
+const Login = () => {
+    const handleSubmit = (evt) => {
```

Comment:
I think this abbreviation is a bit confusing, usually for events we just use `e` or `event`.
Better usually use the whole word, it's always easier to read.

Reply:
Changed it to e!